packages/services/src/inversify.types.ts

```diff
@@ -0,0 +1,4 @@
+export const TYPES = {
```

Comment:
```suggestion
export const Types = {
```

No all caps naming needed here imo, should effectively use enum convention naming, pascal like the properties 🧘🏼‍♂️